content/millennium/dstu2/medications/immunization.md

```diff
@@ -49,10 +49,11 @@ Search for Immunizations that meet supplied query parameters:
 
 ### Parameters
 
- Name      | Required? | Type          | Description
------------|-----------|---------------|-----------------------------------------------------------------------------------------------------
- `patient` | Y         | [`reference`] | The patient for the vaccination record. Example: `12345`
- `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
+ Name      | Required?      | Type          | Description
+-----------|----------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`     | Y or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient` | Y or `_id`     | [`reference`] | The patient for the vaccination record. Example: `12345`
```

Comment:
Let's stick with "This or `other`" to match other parameter tables

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/229/commits/758fc23c2a367f88f960569b578be073cfff6e86